integ-tests/src/main/java/com/google/sql/tablesize.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
```

Comment:
Please add license header.

Reply:
Added everywhere except of sql files because my reader reads them line by line and if I add license info it'll be added to the string.